.github/workflows/e2e-k8s.yml

```diff
@@ -50,6 +50,12 @@ jobs:
     if: ${{ (needs.paths-filter.outputs.not-ignore == 'true') || (github.event_name == 'push') }}
     runs-on: ubuntu-latest
     timeout-minutes: 40
+    strategy:
+      matrix:
+        case:
+          - name: SslShellTaskE2ETest
+            class: org.apache.dolphinscheduler.e2e.cases.ssl.SslShellTaskE2ETest
+
```

Comment:
Why adding this?

Reply:
add an entrance for running E2E in consolidation, k8s.yml, where the running image is a standalone one

Reply:
I don't think this is meaningful for the current e2e-k8s tests. Please remove this modification.